assets/css/translation.css

```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#navcontainerPosition {
+    position: relative;
+}
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#divPullRightHidden {
+    width: 35%;
+}
+
+#faFaGlobe {
```

Comment:
`faGlobe`, and `padding: 5px 0` (can this same style be used for other icons? If yes, it should be a class.)

Reply:
I did not find same padding styles for other icons.Sorry if mistaken.